org.eclipse.capra.ui.plantuml/.gitignore

```diff
@@ -0,0 +1,2 @@
+/bin
+/target
```

Comment:
I had added these directories to the main .gitignore and deleted this file. It worked just fine for me. Any particular reason to reintroduce it?